tests/integrations/smolagents/cassettes/test_smolagents/test_hf_api_model.yaml

```diff
@@ -0,0 +1,717 @@
+interactions:
+- request:
+    body: ''
+    headers:
+      accept:
+      - '*/*'
+      accept-encoding:
+      - gzip, deflate, zstd
+      connection:
+      - keep-alive
+      host:
+      - raw.githubusercontent.com
+      user-agent:
+      - python-httpx/0.28.1
+    method: GET
+    uri: https://raw.githubusercontent.com/BerriAI/litellm/main/model_prices_and_context_window.json
```

Comment:
if possible remove this block

Reply:
@parambharat can we remove this block. Let's get this shipped now and make Soumik happy 😆

Reply:
Removed and fixed this.